src/core/components/layout/components/columns/columns.tsx

```diff
@@ -69,17 +69,12 @@ const Columns = ({
 };
 
 interface ColumnProps extends HTMLAttributes<HTMLDivElement>, Props {
-	width?: number;
+	width?: number | number[];
 	cssOverrides?: SerializedStyles | SerializedStyles[];
 	children: ReactNode;
 }
 
-const Column = ({
-	width = 0,
-	cssOverrides,
-	children,
-	...props
-}: ColumnProps) => {
+const Column = ({ width, cssOverrides, children, ...props }: ColumnProps) => {
```

Comment:
if `width` no longer has a default value of `0`, is this a breaking change?

Reply:
If you were relying on the default before then I think you still get the same behaviour (due to [these changes](https://github.com/guardian/source/pull/726/files#diff-61a31ebdb74d46042d5af57ee18f291987ca2be4bea6ebc25d0ef613c5678e48R141-R142)) but if you're explicitly passing 0 you will now get different behaviour so maybe it's a breaking change? Also, if you pass an empty array I don't think it's applying any styling. Maybe that should be the same as not passing a width?

Reply:
@jamie-lynch is correct, if you were explicitly passing `width={0}` before, the behaviour would have been the same as if you didn't pass a width (i.e. the column's width would be the remainder of container width shared between all zero-width columns). This behaviour was not documented. Arguably the behaviour is bizarre and unexpected.

The new behaviour is to hide a column that has a 0 width. I think this is more intuitive.

Although this is technically a breaking change, I'm inclined not to treat it as such from a semver / release perspective. Nobody is currently relying on this behaviour. Let's close this loophole before people start relying on it.

Reply:
Good catch with the empty array 🙏  

I've updated the logic to treat an empty array the same as passing not passing a width. I hope that makes sense!